Updates to code contribution guide (#4551)
magento/devdocs: _includes/contributor/rewards.md

Achievement | Points | Description
| ------------ | --- | --- |

> New topic | 30 | New topic submissions for content that has never existed on DevDocs such as tutorials, references, instructions, and so on

Reviewer: ```suggestion
New topic | 30 | New topic submissions for content that has never existed in DevDocs, such as tutorials, references, instructions, etc.
```

Major update | 20 | Significant updates to existing content, such as a new section or examples with sample code

> Technical | 10 | Updates to code or processes that alter the technical content of the document such as code snippets, reference documentation, parameter names and values, and so on

Reviewer: ```suggestion
Technical | 10 | Updates to code or processes that alter the technical content of the document, such as code snippets, reference documentation, parameter names and values, etc.
```

> If the PR earns recognition for significant updates and additions to the documentation, it may also receive a special achievement for an additional 20 points.

Reviewer: ```suggestion
If the PR earns recognition for significant updates and additions to the documentation, it may also receive a Special Achievement label, which garners an additional 20 points.
```
